docs/index.md

```diff
@@ -65,7 +65,7 @@ Raises: IOError - File open dialog failed.
 * directory: str - Directory to open file dialog in. Default is the current working directory.
 * default_name: str - Default file name on dialog open. Default is empty.
 * default_ext: str - Default file extension on dialog open. Default is no extension.
-* ext: list[tuple[str, str]] - List of available extensions as (description, extension) tuples. Default is ("All files", "*").
+* ext: list[tuple[str, str]] or list[tuple[str, tuple[str, ...]]] - List of available extensions as (description, extension) or (description, (extensions,)) tuples. Default is ("All files", "*").
```

Comment:
See [this comment](https://github.com/MrThearMan/filedialogs/pull/6/files#r1170169618).